comp/core/status/render_helpers_test.go

```diff
@@ -28,3 +29,40 @@ func TestMkHuman(t *testing.T) {
 	assert.Equal(t, "1", mkHuman("1"))
 	assert.Equal(t, "1.5", mkHuman(float32(1.5)))
 }
+
+func TestParseUnixTime(t *testing.T) {
+	cases := []struct {
+		value          any
+		expectedOutput time.Time
+	}{
+		{int64(1756201396), time.Unix(1756201396, 0)},
+		{int64(1756199835000000000), time.Unix(0, 1756199835000000000)}, // nanoseconds
+		{float64(1756201396.123), time.Unix(1756201396, 0)},
+		{"2025-08-26T11:43:16.000000+02:00", time.Unix(1756201396, 0)},
+	}
+
+	for _, tc := range cases {
+		output, err := parseUnixTime(tc.value)
+
+		assert.WithinDuration(t, tc.expectedOutput, output, 0)
+		assert.Nil(t, err)
```

Comment:
There is a `NoError` helper which has a better output in case of issue, and you might want to only do the second assertion if there was no error (otherwise you'll get two errors in the output, which makes it harder to debug)
```suggestion
		if assert.NoError(t, err) {
			assert.WithinDuration(t, tc.expectedOutput, output, 0)
		}
```